samples/Azure.Storage.Management/readme.md

```diff
@@ -4,7 +4,8 @@
 > see https://aka.ms/autorest
 
 ``` yaml
-title: Azure.Storage.Management
+title: Storage
```

Comment:
So... are we just using folder names to separate things between management and data plane?